packages/postcss/__tests__/samples/panda.config.cjs

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require('@pandacss/dev')
+
+module.exports = defineConfig({
+  preflight: true,
+  presets: ['@pandacss/dev/presets'],
+  include: ['./src/**/*.{ts,tsx,jsx}', './src/**/*.{css,pcss}'],
+  exclude: [],
+  jsxFramework: 'react',
+})
```

Comment:
This might help break the circular dependency

```suggestion
module.exports = {
  preflight: true,
  presets: ['@pandacss/dev/presets'],
  include: ['./src/**/*.{ts,tsx,jsx}', './src/**/*.{css,pcss}'],
  exclude: [],
  jsxFramework: 'react',
}
```